doc/redirects.csv

```diff
@@ -1380,6 +1380,7 @@ generated/pandas.unique,../reference/api/pandas.unique
 generated/pandas.util.hash_array,../reference/api/pandas.util.hash_array
 generated/pandas.util.hash_pandas_object,../reference/api/pandas.util.hash_pandas_object
 generated/pandas.wide_to_long,../reference/api/pandas.wide_to_long
+reference/api/pandas.core.window.Rolling,pandas.core.window.rolling.Rolling.apply
```

Comment:
`Rolling` should not redirect to `Rolling.apply`, but to `Rolling`

Can you also add redirects for the other Rolling pages?

Reply:
Okay I'll be adding a commit soon.
And that review was requested by mistake.

Reply:
But there is nothing like `reference/api/pandas.core.window.rolling.Rolling.html` or `reference/api/pandas.core.window.Rolling.html` should I remove that line?

Reply:
(other redirects work perfectly.)